portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,11 +44,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Get the input from the form.
-    String name = getParameter(request, "name", "anonymous");
-    String comment = getParameter(request, "comment", "");
+    String name = getParameter(request, ENTITY_NAME_HEADER, "anonymous");
+    String comment = getParameter(request, ENTITY_COMMENT_HEADER, "");
```

Comment:
Add the following comment before "anonymous" and "", since it is hard for readers to understand the param.

/* DefaultValue= */

Reply:
Done.